java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JArray.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.java.generator.nodes;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.type.ClassOrInterfaceType;
+
+public class JArray extends AbstractJSONSchema2Pojo {
+
+    private static final String JAVA_UTIL_LIST = "java.util.List";
+
+    private String type = null;
+    private AbstractJSONSchema2Pojo nested = null;
```

Comment:
Could be final, and initialization to `null` removed, IIUC.

Reply:
done